int32 detect_cmd
int32 traffic_light_cmd
